parsec/backend/postgresql/organization.py

```diff
@@ -99,14 +99,32 @@
 """
 )
 
+_q_check_users_limit = Q(
+    f"""
+    SELECT
+        organization._id
+    FROM organization
+    LEFT JOIN
+        user_ on user_.organization = organization._id
+    WHERE
+        organization = { q_organization_internal_id("$organization_id") }
+    GROUP BY organization._id
```

Comment:
Why the `GROUP BY` ? the `WHERE` condition should have left us with only one value for `organization._id`

Reply:
The GROUP BY is mandatory if we use the HAVING condition